src/components/site-form.tsx

```diff
@@ -289,12 +289,27 @@ export const SiteForm = ( {
 						{ onSelectPath && (
 							<>
 								<div className="flex flex-row items-center mb-0">
-									<Button className="pl-0" onClick={ handleAdvancedSettingsClick }>
-										<Icon size={ 24 } icon={ chevronIcon } />
-										<div className="text-[13px] leading-[16px] ml-2">
+									<Button className={ 'pl-0' } onClick={ handleAdvancedSettingsClick }>
```

Comment:
Nitpick: do we need {} if the class name is a string?

Reply:
Yeah I will remove it; I initially planned to use the `cx` function to add red color there but it did not work well is hover states so it is a leftover.

Reply:
Updated in https://github.com/Automattic/studio/pull/564/commits/d5259b0d076bbcfef010ba85733ce8935346660e